azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+name: Instrumented Tests
+trigger:
+- main
+
+
+pool:
+ name: DockerBuildAgents
+
+workspace:
+  clean: all
+steps:
+- script: |
+    docker --version
+    echo =============================================
+    echo Kill all running containers if existing
+    echo =============================================
+    docker container kill $(docker ps -q)
+    echo =============================================
+    echo Run unit and instrumented inside docker container
+    echo =============================================
+    docker run --privileged --cpus="3" --memory="12g" -v "$PWD":/home/gradle/ -w /home/gradle/ authclient.azurecr.io/samples/dbi-instrumented-api30 sh scripts/run-instrumented-tests.sh
```

Comment:
It'd be pretty cool to cache these containers and script their creation.  So when we move to API31, all we need to do is change this number.  Can the API version be a variable?

Reply:
I'm going to check the handlebars templates for the docker files into android-complete, but yes the docker file generation is template and you can change several different variables to produce a different image include: build tools version, api level, type of image (google_apis, default, etc), abi of image, JDK version, etc....

Reply:
The container image is cached in an azure container registry.  Hence the long image name which is a prefix of the azure container registry service endpoint.

Reply:
I'm fine with the long name. ;-)